src/Lavary/Menu/Builder.php

```diff
@@ -556,6 +568,10 @@ public function asUl($attributes = array())
 	 */
 	public function asOl($attributes = array())
 	{
+                if(!empty($attributes['style']) && (($attributes['style'] == 'adminlte') || $attributes['style'] == 'adminLTE')){
```

Comment:
Similar cases here, and at the top. I think adding the ability to include classes on the div is useful, while avoiding the specific AdminLTE implementation.